packages/shared-ui/src/styles/passkey-list.css

```diff
@@ -6,8 +6,10 @@
 }
 
 .cb-passkey-list-title {
-  font-size: calc(var(--cb-base-font-size) * 1.9);
-  margin: 0.5rem 0rem 2rem 0rem;
+  font-size: calc(var(--cb-base-font-size) * 2.7);
```

Comment:
this change is related to PasskeyList, right? Why is it included?

Reply:
This change was to make the UserDetails title consistent in style with PasskeyList title, like shown in Figma